mappings/net/minecraft/entity/boss/dragon/EnderDragonEntity.mapping

```diff
@@ -45,12 +48,14 @@ CLASS net/minecraft/class_1510 net/minecraft/entity/boss/dragon/EnderDragonEntit
 	METHOD method_6819 parentDamage (Lnet/minecraft/class_1282;F)Z
 		ARG 1 source
 		ARG 2 amount
+	METHOD method_6820 getHeadVerticleMovement ()F
```

Comment:
```suggestion
	METHOD method_6820 getHeadVerticalMovement ()F
```
It;s the head and neck's shaking up and down. It's not the movement of a verticle.

Reply:
weird... i click commit for that